readme.rst

```diff
@@ -173,8 +173,8 @@ For example, you might want to add links to announcements, tutorials, getting st
 Configure the spelling check
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-If your documentation uses US English instead of UK English, change this in the
-``.sphinx/spellingcheck.yaml`` file.
+The spelling check uses ``aspell``.
+Its configuration is located in the ``.sphinx/spellingcheck.yaml`` file.
```

Comment:
The fact that aspell is used is superfluous. I would remove that sentence and do:

"The spell check is configured in the ``.sphinx/spellingcheck.yaml`` file."

Followed by the next paragraph ("To add exceptions...").

Then a new one-line paragraph:

"All Canonical products are documented with UK English (``en_GB``)."

Reply:
I don't agree - the fact that the spelling check uses aspell is quite important, both to figure out what configuration options you have (it took me quite a while to find out that there's more than only the few pyspelling options), and to understand that when you run it locally, your local aspell configuration might interfere.

And the paragraph about requiring UK English belongs in the style guide, not the readme for the starter pack. We don't give other rules for how to write your documentation either (like using Diataxis). In theory, the starter pack can be used by anyone, also outside of Canonical. We have configured it according to common practice here, but the starter pack doesn't care about the content (and what language it uses).

Reply:
Okay for point #1. For point #2, people will just change the language to suit them. What this repo is really about is providing consistency across our product sets. We need to agree on the purpose of this repository as similar questions are sure to arise in the future.

Reply:
> For point 2, people will just change the language to suit them. What this repo is really about is providing consistency across our product sets.

But we provide consistency by pre-configuring the starter pack to use UK English?
People can go ahead and change anything - they can put a different logo, change the colours, turn off all automatic checks, and use RST the way they like (for example, use random characters for different headings).

The guidelines for how to create content in the starter pack are in our style guides (for language and for RST/MyST usage).